index.js

```diff
@@ -25,7 +23,7 @@ module.exports = {
 
     const hasMagicallyProvidedQUnit = versionChecker
       .for('ember-qunit')
-      .lt('5.0.0-beta.1');
+      .lt('5.1.4');
```

Comment:
I think this part of the change should be reverted, see my comment on #249 for details

Reply:
fixed! 👍